lib/Echidna/Campaign.hs

```diff
@@ -201,6 +233,8 @@ callseq v w ql = do
       diffs = H.fromList [(AbiAddressType, S.fromList $ AbiAddress . addressWord160 <$> diff)]
   -- Save the global campaign state (also vm state, but that gets reset before it's used)
   hasLens .= snd s
+  when (s ^. _2 . newCoverage) $ addToCorpus res
+  hasLens . ncallseqs += 1  
```

Comment:
shouldn't this be in `randseq`?

Reply:
It seems that if we do that, the effect of increasing the `ncallseqs` is lost. Can we handle this after the merge?

Reply:
sure, as long as we make a ticket